xml/cap_depl_certificates.xml

```diff
@@ -0,0 +1,213 @@
+	    <itemizedlist>
+                <listitem>
+                    Certificates for the SCF router should include: 
+		    <itemizedlist>
+	                <listitem>
+                            *.DOMAIN
+			</listitem>
+		    </itemizedlist>
+		</listitem>
+                <listitem>
+		    Certificates for the  UAA server should include:
+		    <itemizedlist>
+                        <listitem>
+                            uaa.DOMAIN
+			</listitem>
+			<listitem>
+                            *.uaa.DOMAIN
+			</listitem>
+			<listitem>
+                            uaa
+			</listitem>
+                    </itemizedlist>
+            <itemizedlist>
+	</para>
+
+    </sect1>
+
+    <sect1 xml:id="sec.cap.cert-deployment">
+        <title>Deploying Custom Certificates</title>
+	<para>
+            Certificates used in &cap; are configurable through the <filename>values.yaml</filename> files for the deployment of SCF and UAA respectively. To specify a certificate, set the value for the certificate and its corresponding private key under the <literal>secrets:</literal> section using the following properties. 
+        </para>
+	<itemizedlist>
+	    <listitem>
+	        In the <literal>values.yaml</literal> for SCF specify the <literal>ROUTER_SSL_CERT</literal> property and the corresponding <literal>ROUTER_SSL_KEY</literal>.
+	        <screen>
+secrets:
+  ROUTER_SSL_CERT: |
+    -----BEGIN CERTIFICATE-----
+    MIIEEjCCAfoCCQCWC4NErLzy3jANBgkqhkiG9w0BAQsFADBGMQswCQYDVQQGEwJD
+    QTETMBEGA1UECAwKU29tZS1TdGF0ZTEOMAwGA1UECgwFTXlPcmcxEjAQBgNVBAMM
+    CU15Q0Euc2l0ZTAeFw0xODA5MDYxNzA1MTRaFw0yMDAxMTkxNzA1MTRaMFAxCzAJ
+    ...
+    xtNNDwl2rnA+U0Q48uZIPSy6UzSmiNaP3PDR+cOak/mV8s1/7oUXM5ivqkz8pEJo
+    M3KrIxZ7+MbdTvDOh8lQplvFTeGgjmUDd587Gs4JsormqOsGwKd1BLzQbGELryV9
+    1usMOVbUuL8mSKVvgqhbz7vJlW1+zwmrpMV3qgTMoHoJWGx2n5g=
+    -----END CERTIFICATE-----
+
+  ROUTER_SSL_KEY: |
+    -----BEGIN RSA PRIVATE KEY-----
+    MIIEpAIBAAKCAQEAm4JMchGSqbZuqc4LdryJpX2HnarWPOW0hUkm60DL53f6ehPK
+    T5Dtb2s+CoDX9A0iTjGZWRD7WwjpiiuXUcyszm8y9bJjP3sIcTnHWSgL/6Bb3KN5
+    G5D8GHz7eMYkZBviFvygCqEs1hmfGCVNtgiTbAwgBTNsrmyx2NygnF5uy4KlkgwI
+    ...
+    GORpbQKBgQDB1/nLPjKxBqJmZ/JymBl6iBnhIgVkuUMuvmqES2nqqMI+r60EAKpX
+    M5CD+pq71TuBtbo9hbjy5Buh0+QSIbJaNIOdJxU7idEf200+4anzdaipyCWXdZU+
+    MPdJf40awgSWpGdiSv6hoj0AOm+lf4AsH6yAqw/eIHXNzhWLRvnqgA==
+    -----END RSA PRIVATE KEY---- 
+                </screen>
+	    </listitem>
+	    <note>
+		<para>
+                    Note the use of the "|" character which indicates the use of a literal scalar. See the <link xlink:href="http://yaml.org/spec/1.2/spec.html#id2795688">YAML spec</link> for more information.
+		</para>
+	    </note>
+	    <listitem>
+		As certificates will be utilized, the <literal>--skip-ssl-validation</literal> option will not be used when setting a target API endpoint or logging in with the cf CLI. As a result, a certificate will need to be specified for the UAA component as well.
```

Comment:
Delete "As certificates will be utilized"; it doesn't add anything useful. (also "used" is preferable to "utilized")